Documentation/applications/nsh/commands.rst

```diff
@@ -2,8 +2,10 @@
 Commands
 ********
```

Comment:
This should a different headline style if all the following sections should go beneath this level.

Reply:
Done, thanks!